src/manifold/src/sdf.cpp

```diff
@@ -216,24 +201,26 @@ struct ComputeVerts {
         neighborIndex += 1;
         neighborIndex.w = 0;
       }
-      const float val = BoundedSDF(neighborIndex);
+      const float val =
+          voxels[EncodeIndex(neighborIndex + glm::ivec4(1, 1, 1, 0), gridPow)];
```

Comment:
It seems that this `neighborIndex + glm::ivec4(1, 1, 1, 0)` can exceed the `gridSize` range. I see `neighborIndex = glm::ivec4(31, 32, 1, 0)` for `grdSize = glm::ivec3(31, 31, 31)`.

And this line is causing the index out of range error.